media-libs/opencollada/opencollada-1.6.23.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/KhronosGroup/OpenCOLLADA.git"
+inherit multilib cmake-utils
```

Comment:
Another dev told me to do it this way. I didn't like it either, so I am going to stripe out all the live ebuild stuff as I was thinking of removing the live ebuild entirely. It was back when upstream waited like 5 years in between releases, but now they release every time there is a commit. Which means the live ebuild does absolutely nothing. One less headache to worry about. ;)